lib/adiwg/mdtranslator/readers/iso19115_3/iso19115_3_reader.rb

```diff
@@ -10,7 +10,7 @@ module Readers
          module Iso191153
             def self.readFile(file, hResponseObj) # rubocop:disable Naming/MethodName
                # add Iso19115_3 reader version
-               hResponseObj[:readerVersionUsed] = ADIWG::Mdtranslator::Readers::Iso19115_3::VERSION
+               hResponseObj[:readerVersionUsed] = ADIWG::Mdtranslator::Readers::Iso191153::VERSION
```

Comment:
Meant to ask this before. Is this munging of `_3` going to affect merging into the larger project? i.e. to create naming inconsistencies?